pkg/virt-controller/watch/pool/pool.go

```diff
@@ -631,6 +628,17 @@ func indexVMSpec(spec *virtv1.VirtualMachineSpec, idx int) *virtv1.VirtualMachin
 		spec.DataVolumeTemplates[i].Name = indexName
 	}
 
+	appendIndexToConfigMapRefs := false
+	appendIndexToSecretRefs := false
+	if poolSpec.NameGeneration != nil {
```

Comment:
nit - just use `spec` at this point

Reply:
`spec` is the VM template spec, not the pool's spec.

Reply:
My bad sorry!